framework/include/utils/RankTwoTensor.h

```diff
@@ -111,37 +116,124 @@ class RankTwoTensorTempl : public TensorValue<T>
     general = 9
   };
 
-  // Deprecated constructor (replaced by initializeFromRows)
-  RankTwoTensorTempl(const TypeVector<T> & row1,
-                     const TypeVector<T> & row2,
-                     const TypeVector<T> & row3);
+  /**
+   * @brief Initialize the random seed based on an unsigned integer.
+   *
+   * Deprecated in favor of MooseRandom::seed().
+   */
+  static void initRandom(unsigned int);
```

Comment:
Sidebar: this whole global seed business is not great. We should be passing around references to "random number providers". A change in seed here might affect many other systems that don't get the seed reset explicitly, and the seed set here might not even affect future random tensor draws, if some other system overrides the global seed.